src/pages/RoadObstructions/components/DetailFeature/DetailFeature.test.tsx

```diff
@@ -78,23 +74,5 @@ describe('DetailFeature', () => {
     // enable layer
     await user.click(screen.getByLabelText(/wior/i))
     expect(screen.getByLabelText(/wior/i)).toBeEnabled()
-
-    // wior features are loading...
-    await waitFor(() => page.rerender)
-
-    // TODO: find a solution for selecting elements; ideally leaflet features get testid's.
-    try {
-      // wior features are displayed in orange (theme.colors.supplement.orange)
-      // eslint-disable-next-line testing-library/no-container,testing-library/no-node-access
-      const wiorFeatures = page.container.querySelectorAll('.leaflet-overlay-pane svg path[stroke="#ff9100"]')
```

Comment:
Dit hoeft niet meer getest te worden?

Reply:
Ik kreeg het niet werkend. Sowieso is dit geen goede code. Je mag in Jest eigenlijk geen DOM query selectors gebruiken. Ik blijf zeggen: de e2e testen moeten helemaal opnieuw en Amsterdam React Maps moet worden vervangen (is al 2 jaar niet onderhouden).